src/py42/clients/audit_logs.py

```diff
@@ -0,0 +1,41 @@
+class AuditLogsClient(object):
```

Comment:
This file should be named `auditlogs.py` like the other multi-worded files here, such as `securitydata.py` or `detectionlists.py`.

Reply:
Renamed.